nbformat/v4/nbformat.v4.schema.json

```diff
@@ -358,6 +358,10 @@
                     }
                 },
                 "patternProperties": {
+                    "^application\/[a-zA-Z0-9]+[a-zA-Z0-9\\-\\+\\.]+\\+json$": {
+                        "description": "Custom mimetypes with +json suffix",
+                        "type": "object"
+                    },
                     "^(?!application/json$)[a-zA-Z0-9]+/[a-zA-Z0-9\\-\\+\\.]+$": {
```

Comment:
I guess this bit needs to specifically exclude +json mimetypes like it already excludes application/json. I'm just about to travel, though, so hopefully someone else can help with it.